client/app/app.less

```diff
@@ -0,0 +1,183 @@
+@import './common/less/date-picker.less';
+@import './common/less/form.less';
+@import './common/less/global.less';
+@import './common/less/header.less';
+@import './common/less/navbar.less';
+@import './common/less/ovh-angular-input-number-spinner.less';
+@import './common/less/popover.less';
+@import './common/less/revert-really-buttons.less';
+@import './common/less/service.less';
+@import './common/less/tables.less';
+@import './common/less/uiview-animate.less';
+@import './common/less/variables.less';
+@import './common/less/widget-presentation.less';
+@import './telecom/dashboard/telecom-dashboard.less';
+@import './telecom/overTheBox/remote/overTheBox-remote.less';
+@import './telecom/pack/migration/telecom-pack-migration.less';
+@import './telecom/pack/move/pack-move.less';
+@import './telecom/pack/resiliation/pack-resiliation.less';
+@import './telecom/pack/slots/voipLine/activation/pack-voipLine-activation.less';
+@import './telecom/pack/xdsl/access/deconsolidation/pack-xdsl-access-deconsolidation.less';
+@import './telecom/pack/xdsl/access/linediagnostic/pack-xdsl-access-linediagnostic.less';
+@import './telecom/pack/xdsl/access/pack-xdsl-access.less';
+@import './telecom/pack/xdsl/modem/pack-xdsl-modem.less';
+@import './telecom/pack/xdsl/orderFollowUp/pack-xdsl-orderFollowUp.less';
+@import './telecom/sms/sms/compose/addPhonebookContact/telecom-sms-sms-compose-addPhonebookContact.less';
+@import './telecom/telecom.less';
+@import './telecom/telephony/alias/configuration/mode/telecom-telephony-alias-configuration-mode.less';
+@import './telecom/telephony/alias/portability/portabilities/telecom-telephony-alias-portability-portabilities.less';
+@import './telecom/telephony/billingAccount/billing/groupRepayments/telecom-telephony-billing-account-billing-group-repayments.less';
+@import './telecom/telephony/billingAccount/dashboard/telecom-telephony-billing-account-dashboard.less';
+@import './telecom/telephony/billingAccount/manageContacts/telecom-telephony-billing-account-manageContacts.less';
+@import './telecom/telephony/billingAccount/orderAlias/telecom-telephony-billing-account-orderAlias.less';
+@import './telecom/telephony/line/assist/rma/telecom-telephony-line-assist-rma.less';
+@import './telecom/telephony/line/assist/troubleshooting/telecom-telephony-line-assist-troubleshooting.less';
+@import './telecom/telephony/line/calls/click2Call/telecom-telephony-line-calls-click2Call.less';
+@import './telecom/telephony/line/phone/accessories/telecom-telephony-line-phone-accessories.less';
+@import './telecom/telephony/line/phone/configuration/telecom-telephony-line-phone-configuration.less';
+@import './telecom/telephony/service/fax/campaigns/read/telecom-telephony-service-fax-campaigns-read.less';
+@import './telecom/telephony/service/fax/settings/telecom-telephony-service-fax-settings.less';
+@import './telecom/telephony/telecom-telephony.less';
+@import '../components/editable-service-name/editable-service-name.less';
```

Comment:
Can we use same pattern for importing?

`@import './…'` vs `@import '../`.

I personally prefer `./…`.

Reply:
Hum, it depends on the structure, here components is in the parent folder, so it's legit to do this :/

But, if you want, we can manage some aliases, for example: 

```javascript
components: `${process.cwd}/app/components`
```

So, whatever the file where you are, you can use: `components/editable...` as `~` for node_modules for example.